spec/features/toggle_language_spec.rb

```diff
@@ -117,14 +119,14 @@
         expect(find('.pageTitle')).to have_content en_signed_in_root_title
       end
 
-      it 'persists locale selection on a different page' do
+      it 'persists locale selection on a different page', feature: true do
         expect(find('.pageTitle')).to have_content en_signed_in_root_title
         change_language('es')
         expect(find('.pageTitle')).to have_content es_signed_in_root_title
 
         change_page(
           lambda {
-            within '.dashboardNav' do
```

Comment:
The `Dashboard Nav` section is not shown at smaller resolutions. Not sure if this is a bug or not, but the content is still accessible through the main part of the dashboard

Reply:
Yep, that's correct, it's not visible by CSS in smaller resolutions but the content is technically still accessible.

Reply:
these failures here with the dashboard actually aren't related to the header, after getting the parts where accessing the header was working I ran into issues on the dashboards after signing in, let me make a quick gif to show where this was occurring